Utils/Dataflow/017_adjustMetadata/adjustMetadata.py

```diff
@@ -170,23 +175,41 @@ def transform_chain_data(data):
                          'non-dict data: %s. Skipping.\n' % str(data))
         return False
     chain_data = data.get('chain_data')
-    if not chain_data or not chain_data.replace(',', '').isdigit():
+    if not chain_data:
         taskid = data.get('taskid')
         if not taskid:
             sys.stderr.write('(WARN) Function transform_chain_data() '
-                             'received data with incorrect chain_data and '
+                             'received data with empty chain_data and '
                              'without taskid: %s. Skipping.\n' % str(data))
             return False
-        sys.stderr.write('(WARN) Task %s: cannot transform chain_data "%s", '
-                         'it seems to be incorrect. Setting chain_id=%s, '
-                         'chain_data=[%s].\n'
+        sys.stderr.write('(WARN) Task %s: chain_data field is empty.'
+                         'Setting chain_id=%s, chain_data=[%s].\n'
                          % (taskid, chain_data, taskid, taskid))
```

Comment:
Too much arguments for formatting: `chain_data` (its `%s`, to be precise) was removed from text, but not from `% (...)`.

Reply:
My bad, will be fixed.

Reply:
Fixed: 147a34e